Makefile

```diff
@@ -204,6 +220,48 @@ docs/mansidebar.html: docs/mansidebar.html.tt Makefile bin/from-tt
 	@rm -f $@
 	./bin/from-tt releases='master $(MANSERIES)' $<
 
+
+######################################################################
+##
+##  Policy page building sexction
+##
+
+.PHONY: technical-policies
+technical-policies: $(TECHNICAL_POLICIES) bin/md-to-html5 Makefile
+	for x in $(TECHNICAL_POLICIES); do \
+		d=$$(dirname $$x); \
+		f=$$(basename $$x .md); \
+		cat "$$x" \
+			| sed -E -e 's!https?://github\.com/openssl/(general|technical)-policies/blob/master/policies/(.*)\.md!../\1/\2.html!' \
+			| sed -E -e 's!\.\./general/glossary\.html!../glossary.html!' \
```

Comment:
Any reason for the second sed instead of sed -e ... -e ...?
Just stylistic?

Reply:
Let's say that it's just stylistic...